internal/datastore/postgres/migrations/zz_migration.0011_add_xid_constraints.go

```diff
@@ -1,21 +1,9 @@
 package migrations
 
-import (
-	"context"
-	"fmt"
+import "fmt"
 
-	"github.com/jackc/pgx/v4"
-)
-
-const (
-	getNSConfigPkeyName = `
-	SELECT constraint_name FROM information_schema.table_constraints
-		WHERE table_schema = 'public'
-      	AND table_name = 'namespace_config'
-      	AND constraint_type = 'PRIMARY KEY';`
-
-	dropNSConfigIDPkey = "ALTER TABLE namespace_config DROP CONSTRAINT %s;"
-)
+const dropNSConfigIDPkey = `ALTER TABLE namespace_config DROP CONSTRAINT namespace_config_pkey;
```

Comment:
@jakedt I removed the query for the primary key constraint in favor of hardcoding it. What are the scenarios we this constraint name is different? In my tests it was always this value.